src/NuGet.Core/NuGet.Protocol.Core.Types/Model/PackageSearchMetadata.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NuGet.Packaging.Core;
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public class PackageSearchMetadata : IPackageSearchMetadata
```

Comment:
Good point. Current type(s) are incomplete and contain wire protocol or UI specific data. Therefore I couldn't use them for merging as they are.
Planning to refactor 'em by next milestone into a unified class or family of classes.